homeassistant/components/velbus/cover.py

```diff
@@ -66,12 +66,18 @@ def is_closed(self) -> bool | None:
     @property
     def is_opening(self) -> bool:
         """Return if the cover is opening."""
-        return self._channel.is_opening()
+        opening = self._channel.is_opening()
+        if opening:
```

Comment:
Can you use a walrus here?
```suggestion
        if opening := self._channel.is_opening():
```